web/styles/settings.css

```diff
@@ -1919,4 +1938,8 @@ $option_title_width: 180px;
         cursor: default;
         opacity: 0.7;
     }
+
+    .inline {
+        display: inline;
+    }
```

Comment:
How is this class used? I don't see it in the template changes.

Reply:
We have used `settings_checkbox` template for showing the `Users can edit this field` checkbox. So adding this was necessary because `settings_checkbox` template has a class `inline` [here](https://github.com/zulip/zulip/blob/main/web/templates/settings/settings_checkbox.hbs#L8). Without adding this css we would have the following UI.
![Screenshot from 2023-04-04 11-29-03](https://user-images.githubusercontent.com/99073049/229720119-2e81140f-22bf-441b-9502-1829e383f029.png)